libs/accounts/src/lib/accounts-actions-dropdown.spec.tsx

```diff
@@ -0,0 +1,75 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import { AccountsActionsDropdown } from './accounts-actions-dropdown';
+import userEvent from '@testing-library/user-event';
+
+describe('AccountsActionsDropdown', () => {
+  let onClickDeposit: jest.Mock;
+  let onClickWithdraw: jest.Mock;
+  let onClickBreakdown: jest.Mock;
+  let onClickTransfer: jest.Mock;
+
+  beforeEach(() => {
+    onClickDeposit = jest.fn();
+    onClickWithdraw = jest.fn();
+    onClickBreakdown = jest.fn();
+    onClickTransfer = jest.fn();
+  });
+
+  it('should render dropdown items correctly', async () => {
+    // 7001-COLL-005
+    // 7001-COLL-006
+    // 1003-TRAN-001
+    render(
+      <AccountsActionsDropdown
+        assetId="testAssetId"
+        assetContractAddress="testAssetContractAddress"
+        onClickDeposit={onClickDeposit}
+        onClickWithdraw={onClickWithdraw}
+        onClickBreakdown={onClickBreakdown}
+        onClickTransfer={onClickTransfer}
+      />
+    );
+
+    userEvent.click(screen.getByTestId('icon-kebab'));
```

Comment:
If you await the userEvent you won't need to use await screen.findByTestId and can just use screen.getByTestId below

```suggestion
    await userEvent.click(screen.getByTestId('icon-kebab'));
```

Reply:
fixed